### PR TITLE
docs: Root readme improvements

### PR DESCRIPTION
**Pull request by ch4ot1c**
Improvements to the repo's `README.md`

- Grammatical fixes
- Simple English
- Add note for building from source
- ~Add Travis and AppVeyor links~
- Unix as first-class
- Avoid parentheses

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,29 +32,37 @@ The `master` branch is regularly built and tested, but is not guaranteed to be
 completely stable. [Tags](https://github.com/bitcoin/bitcoin/tags) are created
 regularly to indicate new official, stable release versions of Bitcoin Core.
 
-The contribution workflow is described in [CONTRIBUTING.md](CONTRIBUTING.md)
-and useful hints for developers can be found in [doc/developer-notes.md](doc/developer-notes.md).
+The contribution workflow is described in [CONTRIBUTING.md](CONTRIBUTING.md).
+
+Useful hints for developers can be found in [doc/developer-notes.md](doc/developer-notes.md).
+
+Instructions for building Bitcoin Core from source are in [doc/build-unix.md](https://github.com/ch4ot1c/bitcoin/blob/improvements/readme/doc/build-unix.md#to-build).
```
Comment (MeshCollider) [inline README.md]: This refers to your repo

**Comment by MeshCollider** [comment]
IMO this rewrite is a lot to review for small stylistic gain, so I'm a slight NACK here

File: README.md [hunk 2]
```diff
@@ -31,29 +32,37 @@ The `master` branch is regularly built and tested, but is not guaranteed to be
...
 
-There are also [regression and integration tests](/test), written
-in Python, that are run automatically on the build server.
-These tests can be run (if the [test dependencies](/test) are installed) with: `test/functional/test_runner.py`
+Travis CI and AppVeyor CI make sure that every pull request is built for Linux, Windows,
+and macOS. Unit, integration, regression, and sanity tests are run automatically.
 
-The Travis CI system makes sure that every pull request is built for Windows, Linux, and macOS, and that unit/sanity tests are run automatically.
+See https://travis-ci.org/bitcoin/bitcoin and https://ci.appveyor.com/project/DrahtBot/bitcoin
+to view output from the Travis and AppVeyor build servers.
```
Comment (MarcoFalke) [inline README.md]: They are already linked from commits which were run and not linked for commits which are not run. No need to put them into the readme, imo.

**Comment by fanquake** [comment]
Agree with the NAK. Given that this is mostly re-formatting. Going to close for now.

**Pull request closed without merging**